File: 8-Boolean-and-Conditional-Logic/10-Food-Classifying.py
from random import choice
food = choice(['apple', 'grape', 'bacon', 'steak', 'worm', 'dirt'])

if food == 'apple' or food == 'grape':
    print('fruit')
elif food == 'bacon' or food == 'steak':
    print('meat')
else:
    print('yuck')

File: 8-Boolean-and-Conditional-Logic/10-Food-Classifying.js
// custom choice function
function choice(arr) {
  let randomIndex = Math.floor(Math.random() * arr.length)
  return arr[randomIndex]
}

let food = choice(['apple', 'grape', 'bacon', 'steak', 'worm', 'dirt'])

if (food === 'apple' || food === 'grape') {
  console.log('fruit')
}
else if (food === 'bacon' || food === 'steak') {
  console.log('meat')
}
else {
  console.log('yuck')
}
